libs/sdk-ui-dashboard/src/presentation/dragAndDrop/draggableFilterDropZone/DraggableFilterDropZone.tsx

```diff
@@ -72,7 +84,7 @@ export function AttributeFilterDropZone({ targetIndex, onDrop }: AttributeFilter
                         </div>
                     </div>
                     <Bubble alignPoints={[{ align: "bc tc", offset: { x: 0, y: 0 } }]}>
-                        <FormattedMessage id="filterBar.filter.dropzone.tooltip" />
+                        <FormattedMessage id={tooltipId} />
```

Comment:
Isn't this the example that could use `defineMessage`?

Reply:
fixed